kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/KafkaStreamsApp.java

```diff
@@ -112,6 +112,8 @@ protected void doInit() {
             System.exit(1);
           }
       );
+
+      getLogger().info("kafka streams topologies: {}", topology.describe());
```

Comment:
@laxmanchekka Do you want to do at info level?

Reply:
Will this be per stream task?

Reply:
yes. once per kafka streams application at the startup. very useful information to understand the topology.